Library/Formula/kalibrate-rtl.rb

```diff
@@ -0,0 +1,33 @@
+class KalibrateRtl < Formula
+  desc "Tool to Kalibrate RTL devices"
+  homepage "https://github.com/steve-m/kalibrate-rtl"
+  url "https://github.com/steve-m/kalibrate-rtl.git", :revision => "aae11c8a8dc79692a94ccfee39ba01e8c8c05d38"
```

Comment:
It seems like that's not an official upstream, or affiliated with the official upstream at all?